configuration.md

```diff
@@ -0,0 +1,65 @@
+# CNI Configuration
+
+The Calico CNI plugin is configured through the standard CNI [configuration mechanism](https://github.com/appc/cni/blob/master/SPEC.md#network-configuration)
+
+A minimal configuration file that uses Calico for networking and IPAM looks like this
+```json
+{
+    "name": "any_name",
+    "type": "calico",
+    "ipam": {
+        "type": "calico-ipam"
+    }
+}
+```
+
+Additional configuration can be added as detailed below.
+
+## Generic
+### Etcd location
+Specify the location of your etcd cluster using either
+* `etcd_authority` (default is `127.0.0.1:2379`)
+* `etcd_endpoints` (no default. Format is comma separated list of etcd servers e.g. `http://1.2.3.4:2379;http://5.6.7.8:2379`
```

Comment:
Comma separated? Or semicolon separated?  The example says one, the text says another.